src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -92,6 +101,9 @@
 
     private final JsonWebSignature signature = new JsonWebSignature();
 
+    private final Long2ObjectHashMap<Map<String, OAuthAccessGrant>>[] grantsBySubjectByAffinityPerRealm =
```

Comment:
Assign non-static fields in constructor.
`JsonWebSignature` is pseudo-static, acting as a flyweight.